Editor/Scripts/Generators/RosalinaGeneratorHelper.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using UnityEngine;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+
+public static class RosalinaGeneratorHelper
+{
+    public static string Generate(CompilationUnitSyntax compilationUnit, ClassDeclarationSyntax classDeclaration)
```

Comment:
I would be clearer what it's generating if you named it something like `GenerateFile` or `GenerateCompilationUnit`